doc/releases/changelog-0.11.0.md

```diff
@@ -23,175 +27,235 @@
-  [(#1489)](https://github.com/PennyLaneAI/catalyst/pull/1489)
+          for i in range(n):
+              qml.RX(1.5, wires=i)
+
+          return qml.counts()
+
+      return circuit()
+  ```
+
+  ```pycon
+  >>> f(3)
+  (Array([0, 1, 2, 3, 4, 5, 6, 7], dtype=int64),
+  Array([0, 0, 3, 2, 3, 1, 1, 0], dtype=int64))
+  >>> f(10)
+  (Array([ 0,  1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11, 12, 13, 14, 15],      dtype=int64),
+  Array([0, 0, 1, 1, 2, 0, 0, 0, 0, 0, 1, 1, 2, 1, 0, 1], dtype=int64))
+  ```
 
-* Catalyst now supports experimental capture of `cond`, `for_loop` and `while_loop` control flow.
+* Catalyst better integrates with PennyLane program capture, supporting PennyLane-native control flow 
+  operations and providing more efficient transform handling when both Catalyst and PennyLane support 
+  a transform.
   [(#1468)](https://github.com/PennyLaneAI/catalyst/pull/1468)
   [(#1509)](https://github.com/PennyLaneAI/catalyst/pull/1509)
   [(#1521)](https://github.com/PennyLaneAI/catalyst/pull/1521)
+  [(#1544)](https://github.com/PennyLaneAI/catalyst/pull/1544)
+  [(#1561)](https://github.com/PennyLaneAI/catalyst/pull/1561)
+  [(#1567)](https://github.com/PennyLaneAI/catalyst/pull/1567)
+  [(#1578)](https://github.com/PennyLaneAI/catalyst/pull/1578)
 
-  To trigger the PennyLane pipeline for capturing the program as a Jaxpr, simply set
-  `experimental_capture=True` in the qjit decorator.
+  Using PennyLane's program capture mechanism involves setting `experimental_capture=True` in the qjit 
+  decorator. With this present, the following control flow functions in PennyLane are now usable with
+  qjit:
 
-  ```python
-  import pennylane as qml
-  from catalyst import qjit
+  * Support for `qml.cond`:
 
-  dev = qml.device("lightning.qubit", wires=1)
+    ```python
+    import pennylane as qml
+    from catalyst import qjit
 
-  @qjit(experimental_capture=True)
-  @qml.qnode(dev)
-  def circuit(x: float):
+    dev = qml.device("lightning.qubit", wires=1)
 
-      def ansatz_true():
-          qml.RX(x, wires=0)
-          qml.Hadamard(wires=0)
+    @qjit(experimental_capture=True)
+    @qml.qnode(dev)
+    def circuit(x: float):
 
-      def ansatz_false():
-          qml.RY(x, wires=0)
+        def ansatz_true():
+            qml.RX(x, wires=0)
+            qml.Hadamard(wires=0)
 
-      qml.cond(x > 1.4, ansatz_true, ansatz_false)()
+        def ansatz_false():
+            qml.RY(x, wires=0)
 
-      return qml.expval(qml.Z(0))
-  ```
+        qml.cond(x > 1.4, ansatz_true, ansatz_false)()
 
-* Catalyst now supports experimental capture of PennyLane transforms.
-  [(#1544)](https://github.com/PennyLaneAI/catalyst/pull/1544)
-  [(#1561)](https://github.com/PennyLaneAI/catalyst/pull/1561)
-  [(#1567)](https://github.com/PennyLaneAI/catalyst/pull/1567)
-  [(#1578)](https://github.com/PennyLaneAI/catalyst/pull/1578)
+        return qml.expval(qml.Z(0))
+    ```
```

Comment:
for some reason this doesn't register as the end of the codefence on github, leading to the weird rendering below 🤔

Reply:
:notlikethis: